docs/admin/runmetrics.md

```diff
@@ -139,7 +139,7 @@ indicates the number of page faults which happened since the creation of
 the cgroup; this number can never decrease).
 
 
- - **cache:**  
+ - **cache:**
```

Comment:
Oh! Looks like these white-spaces were actually on purpose to force a line-break;

before this change:
<img width="702" alt="screen shot 2016-07-27 at 10 39 18" src="https://cloud.githubusercontent.com/assets/1804568/17169182/8d37e1fa-53e6-11e6-8157-514559f3bff3.png">

after this change:
<img width="595" alt="screen shot 2016-07-27 at 10 39 32" src="https://cloud.githubusercontent.com/assets/1804568/17169186/94f42ebc-53e6-11e6-94ee-fddfbb27ad6f.png">